script/upgrades/MU01/governance/MU01-CGP-Phase2.sol

```diff
@@ -0,0 +1,256 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// solhint-disable func-name-mixedcase, contract-name-camelcase, function-max-lines, var-name-mixedcase
+pragma solidity ^0.5.13;
+pragma experimental ABIEncoderV2;
+
+import { GovernanceScript } from "script/utils/Script.sol";
+import { console2 as console } from "forge-std/Script.sol";
+import { FixidityLib } from "mento-core/contracts/common/FixidityLib.sol";
+
+import { ICeloGovernance } from "mento-core/contracts/governance/interfaces/ICeloGovernance.sol";
+import { IBiPoolManager } from "mento-core/contracts/interfaces/IBiPoolManager.sol";
+import { IPricingModule } from "mento-core/contracts/interfaces/IPricingModule.sol";
+import { Proxy } from "mento-core/contracts/common/Proxy.sol";
+import { Contracts } from "script/utils/Contracts.sol";
+import { Chain } from "script/utils/Chain.sol";
+import { Arrays } from "script/utils/Arrays.sol";
+import { IERC20Metadata } from "mento-core/contracts/common/interfaces/IERC20Metadata.sol";
+
+import { BiPoolManagerProxy } from "mento-core/contracts/proxies/BiPoolManagerProxy.sol";
+import { BrokerProxy } from "mento-core/contracts/proxies/BrokerProxy.sol";
+import { Broker } from "mento-core/contracts/Broker.sol";
+import { BiPoolManager } from "mento-core/contracts/BiPoolManager.sol";
+import { Exchange } from "mento-core/contracts/Exchange.sol";
+import { TradingLimits } from "mento-core/contracts/common/TradingLimits.sol";
+
+import { Config } from "./Config.sol";
+import { ICGPBuilder } from "script/utils/ICGPBuilder.sol";
+
+/**
+ forge script {file} --rpc-url $BAKLAVA_RPC_URL 
+                     --broadcast --legacy 
+ * @dev depends on: ../deploy/*.sol
+ */
+contract MU01_CGP_Phase2 is ICGPBuilder, GovernanceScript {
+  using TradingLimits for TradingLimits.Config;
+
+  ICeloGovernance.Transaction[] private transactions;
+
+  Config.PoolConfiguration private cUSDCeloConfig;
+  Config.PoolConfiguration private cEURCeloConfig;
+  Config.PoolConfiguration private cBRLCeloConfig;
+  Config.PoolConfiguration private cUSDUSDCConfig;
+  Config.PoolConfiguration[] private poolConfigs;
+
+  address private cUSD;
+  address private cEUR;
+  address private cBRL;
+  address private celo;
+  address private bridgedUSDC;
+
+  address private cUSDUSCDRateFeedId = address(uint256(keccak256(abi.encodePacked("USDCUSD"))));
```

Comment:
I'm surprised this doesn't throw a compile error, I get an error when trying it in the REPL:
```
➜ address(uint256(keccak256(abi.encodePacked("USDCUSD"))));
Compiler errors:
error[9640]: TypeError: Explicit type conversion not allowed from "uint256" to "address".
  --> ReplContract.sol:14:9:
   |
14 |         address(uint256(keccak256(abi.encodePacked("USDCUSD"))));
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

since it should be this instead:

```
➜ address(uint160(uint256(keccak256(abi.encodePacked("USDCUSD")))))
Type: address
└ Data: 0xa1a8003936862e7a15092a91898d69fa8bce290c
```

but anyways, it's not being used, so lets remove it?